plugins/tiddlywiki/multiwikiserver/modules/store/sql-tiddler-store.js

```diff
@@ -151,8 +151,9 @@ SqlTiddlerStore.prototype.processIncomingTiddler = function(tiddlerFields) {
 	const attachmentsEnabled = this.adminWiki.getTiddlerText("$:/config/MultiWikiServer/EnableAttachments","yes") === "yes";
 	const contentTypeInfo = $tw.config.contentTypeInfo[tiddlerFields.type || "text/vnd.tiddlywiki"],
 		isBinary = !!contentTypeInfo && contentTypeInfo.encoding === "base64";
-	if(attachmentsEnabled && isBinary && tiddlerFields.text && tiddlerFields.text.length > attachmentSizeLimit) {
-		const attachment_blob = this.attachmentStore.saveAttachment({
+
```

Comment:
We generally avoid blank lines within function definitions unless they are indispensable for readability.

Reply:
I intentionally put blank line and space, but let me try to follow the convention of the project.